File: preview/deployed_devices/device/update-update-device/update-update-device.5.x.php
<?php

// Update the path below to your autoload.php,
// see https://getcomposer.org/doc/01-basic-usage.md
require_once '/path/to/vendor/autoload.php';

use Twilio\Rest\Client;

// Find your Account Sid and Auth Token at twilio.com/console
$sid    = "ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX";
$token  = "your_auth_token";
$twilio = new Client($sid, $token);

$device = $twilio->preview->deployedDevices->fleets("FLXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX")
                                           ->devices("THXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX")
                                           ->update(array(
                                                        "friendlyName" => "My New Device",
                                                        "identity" => "bob@twilio.com"
                                                    )
                                           );

print($device->uniqueName);

File: preview/deployed_devices/device/update-update-device/update-update-device.5.x.rb
# Download the helper library from https://www.twilio.com/docs/ruby/install
require 'rubygems'
require 'twilio-ruby'

# Your Account Sid and Auth Token from twilio.com/console
# DANGER! This is insecure. See http://twil.io/secure
account_sid = 'ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX'
auth_token = 'your_auth_token'
@client = Twilio::REST::Client.new(account_sid, auth_token)

device = @client.preview.deployed_devices
                        .fleets('FLXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX')
                        .devices('THXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX')
                        .update(
                           friendly_name: 'My New Device',
                           identity: 'bob@twilio.com'
                         )

puts device.unique_name
